### PR TITLE
Gi kl/layout

### PR DESCRIPTION
**Pull request by gennadiyI**
Layout Created

### CODE REVIEW

**Comment by gennadiyI** [comment]
Wrong pull request branch (master denied)

File: src/components/Main/Main.js [hunk 1]
```diff
@@ -0,0 +1,44 @@
...
+            </Card>
+            <Card body className="paddingBottom">
+            <Row>
+            <Col xs="auto" className="positioning">
+              <CardTitle>Special Title Treatment</CardTitle>
+              <CardText>With supporting text below as a natural lead-in to additional content.</CardText>
+              <Button>Go somewhere</Button>
+              </Col>
+              </Row>
+
```
Comment (janisknets) [inline src/components/Main/Main.js]: unneeded newlines

File: src/App.js [hunk 2]
```diff
@@ -1,15 +1,31 @@
 import React, { Component } from "react";
 import "./App.css";
+
+// import Layout from "./components/Layout/Layout"
```
Comment (rokija) [inline src/App.js]: remove commented out lines

File: src/App.js [hunk 3]
```diff
@@ -1,15 +1,31 @@
...
+// import Registration from "./components/Registration/Registration"
```
Comment (janisknets) [inline src/App.js]: don't keep unused commented lines

**Review by janisknets — CHANGES_REQUESTED** [review]

File: src/App.js [hunk 4]
```diff
@@ -1,15 +1,31 @@
...
 
+
 class App extends Component {
   render() {
-    return (
-      <React.Fragment>
-        <Header />
-      </React.Fragment>
+      return (
+          <div className="appSite">
```
Comment (rokija) [inline src/App.js]: please use naming according to BEM
http://getbem.com/naming/

in this case, main className should be "App",
content should be "App__content", etc..

File: src/App.js [hunk 5]
```diff
@@ -1,15 +1,31 @@
...
+              <div className="siteContent">
+                  <div className="appHeader">
+                  <React.Fragment>
```
Comment (rokija) [inline src/App.js]: why us React.Fragment needed here?

File: src/App.js [hunk 6]
```diff
@@ -1,15 +1,31 @@
...
+                     <Header />
+                   </React.Fragment>
+                  </div>
+                  <div className="main">
```
Comment (rokija) [inline src/App.js]: fix className naming (using BEM)

File: src/components/Footer/Footer.js [hunk 7]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import "./Footer.css"
+
+
+class Footer extends Component {
+    render() {
+        return(
+    <footer>
```
Comment (rokija) [inline src/components/Footer/Footer.js]: remove <footer>, div below is enough

File: src/components/Footer/Footer.js [hunk 8]
```diff
@@ -0,0 +1,17 @@
+import React, { Component } from "react";
+import "./Footer.css"
+
+
```
Comment (rokija) [inline src/components/Footer/Footer.js:3]: remove empty line

File: src/components/Footer/Footer.js [hunk 9]
```diff
@@ -0,0 +1,17 @@
...
+class Footer extends Component {
+    render() {
+        return(
+    <footer>
+      <div className="footerComponent">
```
Comment (rokija) [inline src/components/Footer/Footer.js]: rename to Footer (main div in the component should be the same as the name of the component, in this case Footer)

File: src/components/Main/Main.css [hunk 10]
```diff
@@ -0,0 +1,25 @@
+/* // CSS styling. Should decide what goes there (Background, colors etc) */
+.divMain {
+    margin-top: 10em;
+}
+
+.Layout {
+    background-color: #8b9dc3;
+    
+
```
Comment (rokija) [inline src/components/Main/Main.css]: remove empty lines

File: src/components/Main/Main.css [hunk 11]
```diff
@@ -0,0 +1,25 @@
...
+.Layout {
+    background-color: #8b9dc3;
+    
+    
+    }
+
+.positioning {
+    margin:auto;
+
```
Comment (rokija) [inline src/components/Main/Main.css]: remove empty line

File: src/components/Main/Main.css [hunk 12]
```diff
@@ -0,0 +1,25 @@
...
+.positioning {
+    margin:auto;
+    
+}
+
+.paddingBottom {
+    margin-bottom: 3em;
+}
+
+/* .blockBackground {
```
Comment (rokija) [inline src/components/Main/Main.css]: remove commented out code

File: src/components/Main/Main.js [hunk 13]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import   "./Main.css"
+// import Registration from "../Registration/Registration"
```
Comment (rokija) [inline src/components/Main/Main.js]: remove commented out code

File: src/components/Main/Main.js [hunk 14]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import   "./Main.css"
```
Comment (rokija) [inline src/components/Main/Main.js]: remove  empty spaces

File: src/components/Main/Main.js [hunk 15]
```diff
@@ -0,0 +1,44 @@
...
+import { Container, Row, Col } from 'reactstrap';
+import { Card, Button, CardTitle, CardText } from 'reactstrap';
+
+ class Main extends Component {
+   render() {
+     return(
+        <div className="divMain">
+       <Container>
+           
+
```
Comment (rokija) [inline src/components/Main/Main.js]: remove empty lines

File: src/components/Main/Main.js [hunk 16]
```diff
@@ -0,0 +1,44 @@
+import React, { Component } from "react";
+import   "./Main.css"
+// import Registration from "../Registration/Registration"
+import { Container, Row, Col } from 'reactstrap';
+import { Card, Button, CardTitle, CardText } from 'reactstrap';
+
+ class Main extends Component {
+   render() {
+     return(
+        <div className="divMain">
```
Comment (rokija) [inline src/components/Main/Main.js]: className should be "Main"

File: src/components/Main/Main.js [hunk 17]
```diff
@@ -0,0 +1,44 @@
...
+              <CardText>With supporting text below as a natural lead-in to additional content.</CardText>
+              <Button>Go somewhere</Button>
+              </Col>
+              </Row>
+              
+            </Card>
+
+            </div>
+
+
```
Comment (rokija) [inline src/components/Main/Main.js]: empty lines...

**Review by rokija — CHANGES_REQUESTED** [review]
please add human readable titles to pull requests (instead of Gi kl/layout)

**Review by janisknets — APPROVED** [review]

**Review by rokija — APPROVED** [review]

**Pull request merged**